### PR TITLE
Show loading div while waiting for results of preferences lookup

### LINKED ISSUE #110: Surveys page on organizer app shows "disabled" on each load
When loading the survey page in the organizer app, the message "Surveys have been disabled, please check your preferences" is always shown for about a second before surveys load.

### PR DESCRIPTION
**Pull request by StevenLocke**
@Calligre/frontend 
Fixes #110

### CODE REVIEW

File: src/js/pages/organizer/Surveys.jsx [hunk 1]
```diff
@@ -41,7 +42,9 @@ export default class Surveys extends React.Component {
   }
 
   loadPreferences = () => {
-    this.setState({ disabled: !PreferenceStore.preferences.survey });
+    this.setState({ disabled: !PreferenceStore.preferences.survey,
```
Comment (larajanecka) [inline src/js/pages/organizer/Surveys.jsx]: new line for each attribute in object

**Review by larajanecka — APPROVED** [review]
I suspect this code is going to be all over the place in the near future. We might want to look at modularizing it.

File: src/js/pages/organizer/Surveys.jsx (hunk 1 above)
Comment (StevenLocke) [reply src/js/pages/organizer/Surveys.jsx]: I don't follow. There is currently a new line between attributes, should it all be the same line?

Comment (larajanecka) [reply src/js/pages/organizer/Surveys.jsx]: Sorry, I meant either a new line for all attributes as in the disabled attribute should get a new line after the setstate. Its a small nitpick. Did the linter not catch this?

Comment (StevenLocke) [reply src/js/pages/organizer/Surveys.jsx]: Nope, but changed that now.

**Comment by ericljpemberton** [comment]
:+1: Resolve conflicts and merge?

**Pull request merged**